src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -290,28 +299,7 @@ public void repeat() throws SpeechletException, URISyntaxException, IOException
         // Try the repeat intent - this simulates after initial setup, when we don't yet have anything to repeat
         SpeechletResponse sr = authedSpeechlet.onIntent(repeatIntent, session);
         String spoken = ((PlainTextOutputSpeech)sr.getOutputSpeech()).getText();
-        assertThat(spoken, containsString("I'm sorry, I don't have anything to repeat.  You can ask me for arrival times for your stop."));
-
-        ObaArrivalInfo[] obaArrivalInfoArray = new ObaArrivalInfo[1];
-        obaArrivalInfoArray[0] = obaArrivalInfo;
-        new Expectations() {{
-            obaArrivalInfo.getShortName(); result = "8";
-            obaArrivalInfo.getHeadsign(); result = "Mlk Way Jr";
-            obaArrivalInfoResponse.getArrivalInfo(); result = obaArrivalInfoArray;
-            obaUserClient.getArrivalsAndDeparturesForStop(anyString, anyInt); result = obaArrivalInfoResponse;
-        }};
-        String response = "Route 8 Mlk Way Jr is now arriving based on the schedule -- ";
-
-        // Test initial request/response
-        sr = authedSpeechlet.onLaunch(launchRequest, session);
-        spoken = ((PlainTextOutputSpeech)sr.getOutputSpeech()).getText();
-        assertThat(spoken, equalTo(response));
-
-        // Now try repeat intent again - we should get the last response again
-        session.setAttribute(PREVIOUS_RESPONSE, response);  // FIXME - this should be set via the above call, but isn't?
-        sr = authedSpeechlet.onIntent(repeatIntent, session);
-        spoken = ((PlainTextOutputSpeech)sr.getOutputSpeech()).getText();
-        assertThat(spoken, containsString(response));
+        assertThat(spoken, equalTo(prevResponse));
```

Comment:
So to clarify - the implementation of repeat intent in the project works fine, it's the test itself that doesn't perform as anticipated.  I'm assuming it's due to me not setting up the test correctly.  I'd like to keep my original test design, but figure out how to eliminate the call to `session.setAttribute(PREVIOUS_RESPONSE, response);`, and let the actual implementation set this internally within `sr = authedSpeechlet.onLaunch(launchRequest, session);`.